### DISCUSSION TITLE
CI: Ubunut 18.04 brownout

### DISCUSSION THREAD

**Discussion by osteffenrh** [tianocore/edk2#3479] [General]
The CI still uses Ubuntu 18.04 images, which will disappear December 1st.

It is already starting to make problems, see https://dev.azure.com/tianocore/edk2-ci/_build/results?buildId=65295&view=results

![image](https://user-images.githubusercontent.com/95756456/195858326-ae3e3ac1-186a-489f-a632-556fdf7313b2.png)

I tried just bumping the image up to 20.04 or even 22.04, but there are some other issues with that.
(It seems qemu is missing.)

Just want to say we should do something soon.

**Comment by spbrogan** [comment]
My suggestion is to get the containers build patch set finalized and approved before the 1st and then switch over to that.

**Reply by osteffenrh** [reply]
Agreed.

In case we need more time to find a good solution for the ext_dep question, I'd say we can also propose the patch set without the commit that removes the ext_dep files. In that case the CI will still download the toolchain at runtime, but at least it would be running on an image that we control.

I am currently rebasing the series to the latest master and the latest container image. Will post on the mailing list.
